### PR TITLE
Fix bower management command for Django 1.10

### PR DESCRIPTION
**Pull request by freeyoung**

### CODE REVIEW

**Comment by PetrDlouhy** [comment]
@freeyoung There is also bower_install command, which fails with following exception:
```  
File "/home/petr/soubory/programovani/Auto-mat/aklub/aklub/env/src/django-bower/djangobower/management/commands/bower_install.py", line 8, in Command
    option_list = BaseBowerCommand.option_list + (
AttributeError: type object 'BaseBowerCommand' has no attribute 'option_list'
```

**Comment by PetrDlouhy** [comment]
@freeyoung: Could you please fix the tests (merging with current master might be enough) and add Django 1.10 to the .travis.yml?

**Comment by freeyoung** [comment]
@PetrDlouhy Sure! I will give it a look today.

**Comment by freeyoung** [comment]
@PetrDlouhy I've fixed the tests for django >= 1.8. However for lower versions of django I could not really fix the tests because django 1.8 [changed the way to implement custom management commands](https://docs.djangoproject.com/en/1.10/releases/1.8/#management-commands).

Now that django versions < 1.8 have reached the end of support since December 2015, I propose to remove these versions from `.travis.yml`. What do you think?

**Comment by PetrDlouhy** [comment]
I am not maintainer of this repository, so we should ask @nvbn if it is possible to drop the support.

File: djangobower/management/base.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def _check_bower_exists(self):
         if not bower_adapter.is_bower_exists():
             raise BowerNotInstalled()
 
-    def _install(self, args):
+    def _install(self, args=[]):
```
Comment (Behoston) [inline djangobower/management/base.py]: It should be:
```
def _install(self, args=None):
    if args is None:
        args = []
```

**Comment by yprez** [comment]
+1, supporting 1.4 is a waste of time imo...

**Comment by freeyoung** [comment]
Since #71 has been merged, closing.

**Pull request closed without merging**